spirv-tools.yaml

```diff
@@ -23,7 +23,7 @@ environment:
 pipeline:
   - uses: fetch
     with:
-      expected-sha256: 1c1d428537031ff0a753a6ebcc473e9c2b39bda351b556f5378cab1c0cf9c278
+      expected-sha256: 6cfa228695e4a0300ff30eafd88056128c67342a0f0838400cb3a566caadc7d4
```

Comment:
testing this locally now

Reply:
We need a `spirv-headers` checksum change because they retagged things upstream.

Reply:
This is fixed in the PR linked below which subsumes this change